clients/linux/src/app.rs

```diff
@@ -603,7 +603,18 @@ impl LbApp {
                     d.close();
                     let acctscr = &lb.gui.account;
                     acctscr.sidebar.tree.set_name(&id, &name);
-                    lb.messenger.send(Msg::RefreshSyncStatus);
+
+                    match lb.core.file_by_id(meta.id) {
+                        Ok(renamed_meta) => {
+                            lb.gui.win.set_title(&renamed_meta.name);
+                            lb.gui.account.set_search_field_text(&lb.core.full_path_for(&renamed_meta));
+                            lb.messenger.send(Msg::RefreshSyncStatus);
+                        }
+                        Err(err) => {
+                            d.close();
```

Comment:
I don't think we need to `d.close()` here? We do so [on 603 above](https://github.com/lockbook/lockbook/pull/705/files#diff-c333b08e1a270da5874d2c32f455ac27d18f7854b40b6d733d50e8e6439347c7R603) before the match statement.